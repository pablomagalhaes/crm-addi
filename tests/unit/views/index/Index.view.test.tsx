import React from 'react';
import { shallow } from 'enzyme';

import IndexView from '@/views/index/Index.view';

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
}));

describe('<IndexView />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<IndexView />);
    expect(wrapper).toMatchSnapshot();
  });
});
