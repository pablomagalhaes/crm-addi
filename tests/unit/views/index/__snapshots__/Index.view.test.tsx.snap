// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexView /> should match snapchat 1`] = `
<Fragment>
  <LeadsSection />
  <ProspectorsSection />
</Fragment>
`;
